### ISSUE TITLE
input switch transition

### ISSUE THREAD

**Issue by ThaUnknown** [halfmoonui/halfmoon#42]
would be nice to add a simple transition to the checkmark switch like so:
```css
.custom-switch input[type=checkbox]:checked~label:after,
.custom-switch input[type=checkbox]~label:after{
    transition: left .2s ease
}
``` 
it's really simple and makes it feel a LOT better

**Comment by halfmoonui** [comment]
Thank you, I will test this out and see how it works.

**Comment by halfmoonui** [comment]
@ThaUnknown This has been fixed in v1.1.1: https://www.gethalfmoon.com/docs/switch

Though I did change the transition speed slightly, the faster one just looked nicer to me. I will go ahead and close this issue. You can re-open it if you think the issue is still not resolved, or open another issue. Thank you.
